hw/ip/sram_ctrl/doc/_index.md

```diff
@@ -163,8 +163,8 @@ The different configuration options are listed in the table below:
 
  EN_SRAM_IFETCH (OTP) | HW_DEBUG_EN (Life Cycle) | EXEC CSR | Execution Enabled
 ----------------------|--------------------------|----------|--------------------
- == 0xA5              | -                        | == 0x5   | Yes
- == 0xA5              | -                        | != 0x5   | No
+ == 0xA5              | -                        | == 0xA   | Yes
```

Comment:
Done, only for consistency I used `kMubi4True` and `kMubi8True`, instead of `kMultiBitBool4True`, `kMultiBitBool8True`. Constant names as found in the RTL rather than `multibits.h`. I, however, don't have preference and happy to change if you guys prefer the SW version.

Reply:
Wouldn't you want these to be the SW enums defined in `multibits.h`? Since most users reading these docs will be using SW to access the sram_ctrl EXEC CSR?

Reply:
Not true - DV engineers heavily rely on the spec too - they may or may not care about any piece of SW code. 
Lets just align SW and RTL enums to have the same literals (minus the `k` prefix in RTL)?

Reply:
i think if we are to align, i would probably align to `multibits.h`.
mostly because a generic user of the chip / fpga won't really know anything about the contents of `prim_mubi_pkg`, but they can and likely will find `multibits.h` in the sw area and in our DIFs.

Reply:
Done, used values as in `multibits.h`, please note that we may or may not want to change the `.hjson`, as it references the RTL style naming (see `sram_ctrl.EXEC ` description). This is a nit, which if deemed worthy - I can fix in a separate PR.

I guess the bigger question is, shall we be using the SW names for multibit values in the register descriptions across all peripherals? @tjaychen , what do you think?

Reply:
yes that is the plan! i'm going to update them as i come across them (i have been doing that for the entropy blocks / clkmgr)